signup-service/src/main/java/io/mosip/signup/services/RegistrationService.java

```diff
@@ -251,7 +251,7 @@ public RegisterResponse register(RegisterRequest registerRequest, String transac
         transaction.setRegistrationStatus(RegistrationStatus.PENDING);
         cacheUtilService.setStatusCheckTransaction(transactionId, transaction);
 
-        notificationHelper.sendSMSNotificationAsync(registerRequest.getUserInfo().getPhone(), transaction.getLocale(),
+        notificationHelper.sendSMSNotificationAsync(registerRequest.getUserInfo().getPhone(), registerRequest.getLocale(),
```

Comment:
if the language code is null in the request, we can take the language code from the transaction. what do you think @mengleang-0090?